test/gbi.h

```diff
@@ -0,0 +1 @@
+#include "../libgfxd/gbi.h"
```

Comment:
Could an include path for libgfxd be specified in the makefile rather than this dummy file?

Reply:
The idea with this wrapper .h would also be to let someone try another gbi.h, since libgfxd's is glank's i.e. not sgi's / decomp's

But yeah this can be changed idk